plugins/catalog/src/components/EntitySwitch/EntitySwitch.tsx

```diff
@@ -59,6 +62,7 @@ type SwitchCaseResult = {
  */
 export interface EntitySwitchProps {
   children: ReactNode;
+  render?: 'first' | 'all';
```

Comment:
Let's bikeshed the name a little. What if we extend with more behaviors later, such as rendering nothing (it essentially being a failure if there's more than one match) or the fallback. I kinda want the word "multiple" in the name or something like that. Maybe `renderMultipleMatches?: 'first' | 'all'`, perhaps?